src/ralph/admin/widgets.py

```diff
@@ -212,14 +213,14 @@ def render(self, name, value, attrs=None):
         can_edit = self.admin_site._registry[self.rel_to].has_change_permission(
             self.request
         )
-        if value and can_edit:
+        if value and can_edit and not issubclass(self.rel_to, Polymorphic):
             context['change_related_template_url'] = self.get_related_url(
                 info, 'change', value,
             )
         can_add = self.admin_site._registry[self.rel_to].has_add_permission(
             self.request
         )
-        if can_add:
+        if not issubclass(self.rel_to, Polymorphic) and can_add:
```

Comment:
check my previous comment above - we should somehow distinguish final classes (`DataCenterAsset`, `VIP`) etc from base classes (`BaseObject`, `Asset`)